marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/entity-lib.sjs

```diff
@@ -341,9 +341,9 @@ function writeModelToDatabases(entityName, model, databases) {
     // Using xdmp.invoke results in e.g. 20 models being saved in several seconds as opposed to well under a second
     // when calling xdmp.documentInsert directly.
     if (db === xdmp.databaseName(xdmp.database())) {
-      xdmp.documentInsert(entityLib.getModelUri(entityName), model, permissions, getModelCollection());
+      xdmp.documentInsert(getModelUri(entityName), model, permissions, getModelCollection());
```

Comment:
I don't know why entityLib was here before - it's not needed since these are all function calls within entity-lib